src/config.template.json

```diff
@@ -1,6 +1,15 @@
 {
+  "title": "Custom vitrivr",
+  "api": {
+    "host": "vitrivr-domain.com",
+    "port": "1234"
+  },
+  "resources": {
+    "host_thumbnails": "http://localhost:4567/thumbnails/:s",
+    "host_objects": "http://localhost:4567/objects/:o"
+  },
   "competition": {
-    "host": "",
+    "host": "dres.org",
```

Comment:
I suggest to also use `dres-domain.com` or similar, as `dres.dev` is DRES's official home (which does point to its github page, and not to a valid endpoint).